app/src/main/java/com/devshish/internship/presentation/ui/lyrics/LyricsFragment.kt

```diff
@@ -25,6 +28,19 @@ class LyricsFragment : Fragment(R.layout.fragment_lyrics) {
         super.onViewCreated(view, savedInstanceState)
         setHasOptionsMenu(true)
 
+        with(binding) {
+            topAppBar.apply {
+                setupWithNavController(findNavController())
+                title = args.searchSong.title
+                subtitle = args.searchSong.artist
```

Comment:
Setup this song from viewmodel, and not from arguments 
Arguments in our case only need to instantiate viewmodel, and then viewmodel decides how and what to show